scripts/verify.sh

```diff
@@ -175,8 +175,8 @@ source $SCRIPTDIR/common.bash
 
 header
 log separator
-verify_min_k8s_version "1" "20"
-verify_max_k8s_version "1" "25"
+verify_min_k8s_version "1" "21"
```

Comment:
Revert this change as operator supports n-2 version of the driver.

Reply:
@rensyct - Powerscale driver should support K8s 1.22 version for VISA. Either we can set the min K8s version to 1.22 but not sure anyone using K8s 1.21, so i didn't made any change to it.

Reply:
@rajkumar-palani , dell-csi-operator supports n-2 versions of the driver. Previous versions of the driver support K8s 1.21. So please revert the changes made to K8s min version from 21 to 20. Line No 178

Reply:
@rensyct - Reverted the change, as discussed.